rainier-compute/src/main/scala/com/stripe/rainier/compute/Cholesky.scala

```diff
@@ -0,0 +1,86 @@
+package com.stripe.rainier.compute
+
+/*
+This represents an NxN matrix A
+by the  N(N+1)/2 packed lower-triangular elements
+of its Cholesky decomposition L
+(where A = LL*)
+*/
+case class Cholesky(packed: Vector[Real]) {
```

Comment:
to be clear, packed is `L` here or `L*`?

Reply:
packed is `L`